oci/layout/oci_dest_test.go

```diff
@@ -133,6 +133,27 @@ func TestPutTwoDifferentTags(t *testing.T) {
 	assert.Equal(t, "zomg", index.Manifests[2].Annotations[imgspecv1.AnnotationRefName])
 }
 
```

Comment:
I think you should point out here that the substantial part of the patch was already applied (to oci/layout/oci_dest.go) and you are only adding the test.

also please add a 'cherrypicked from' line as 'git cherry-pick -x' would have given you